import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -67,9 +68,9 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
     JCommander jCommander = new JCommander(flags);
     jCommander.parse(args);
 
-    String dicomwebAddress = DicomWebValidation.validatePath(flags.dicomwebAddress, DicomWebValidation.DICOMWEB_ROOT_VALIDATION);
```

Comment:
I think users should still have the option to pass in the dicomweb address if they want.

Reply:
done